### PR TITLE
Add Figma documentation

### PR DESCRIPTION
**Pull request by ZainKho**
This pull request adds documentation and content for the Figma design tools page. A few notes I wrote down about creating GIFs and images for the docsites:

### GIF and Image Guidelines
- Should not be longer than 10 seconds
- 50% width should be 800 x 500 px (or proportional up to 100% width)
    - Crop and convert from mov to gif using [EZGif](https://ezgif.com)
- 100% width should be 932 x 383 px (or proportional up to 100% width)
    - Crop using [EZGif](https://ezgif.com)
    - Convert to gif using [Zamzar](https://www.zamzar.com/convert/mov-to-gif/)

### Figma Live Embed Guidelines
- Should only be used when one page is necessary to understand the document
- Dimensions should be 1600 x 1000 px (or proportional up to 100% width)

I'm looking for feedback on copy and content. If anyone has suggestions, I greatly appreciate it!

### CODE REVIEW

File: pages/design/tools/figma.mdx [hunk 1]
```diff
@@ -1,6 +1,45 @@
...
+    <Box width={12/12}>
+      <BorderBox height="250px" overflow='hidden'>
+       <img src="https://user-images.githubusercontent.com/6846673/61730055-6ddf6000-ad2d-11e9-92ce-cd14fccc6851.png"/>
+      </BorderBox>
+    </Box>
+   </Flex>
+</Text>
+
+## Styles
+GitHub’s Figma styles help give our products the same great feel across multiple products, sites, and places.
```
Comment (ashygee) [inline pages/design/tools/figma.mdx]:
```suggestion
Primer uses Figma styles to maintain a consistent look and feel across all of GitHub's UI. We have provided styles for color variables as well as our typography.
```

File: pages/design/tools/figma.mdx [hunk 2]
```diff
@@ -1,6 +1,45 @@
...
+      </BorderBox>
+      <Text textAlign='center' color='gray.6'>The type system includes bold, uppercase, and small variations for body and header text.</Text>
+    </Box>
+   </Flex>
+</Text>
+
+Check out our [color guidelines](https://primer.style/design/foundation/color) and [typography guidelines](https://primer.style/design/foundation/typography) for more details.
 
 ## Components
+Components are elements that have a common visual style and purpose across GitHub products. While they may share the same basic design, some have hidden layers or groups for different states or colors. In Figma, you can access these components from the “Assets” tab at the top of the lefthand sidebar. Here's a list of our current components (and their Figma files):
```
Comment (ashygee) [inline pages/design/tools/figma.mdx]:
```suggestion
Primer uses components for all of our common UI patterns. For components that have several different states, we use layers or groups within each component to quickly switch between states.

In Figma, you can access these components from the “Assets” tab at the top of the lefthand sidebar. Here's a list of our current components (and their Figma files):
```

File: pages/design/tools/figma.mdx [hunk 3]
```diff
@@ -24,12 +63,42 @@
 - [Tooltips](https://www.figma.com/file/EmivQqEsAexN5eEclS78JTVH/primer-tooltips)
 
 
-## Utilities
-- [Colors](https://www.figma.com/file/IbJhpJJZRwGg4KlBE6TxhaQn/primer-colors)
-- [Cursors](https://www.figma.com/file/MbUdjMu8XZWBepwihiDFaFtT/Cursors)
-- [GitHub Templates](https://www.figma.com/file/VCkfDFhhJQz32TQSxGyLntrt/primer-buttons)
-- [Typography](https://www.figma.com/file/gHUvCi7ypiw9d2VzgqfhchNH/primer-typography)
+## Prototyping and Wireframing
+We’ve developed a set of page templates to help you get started with exploration and prototyping in Figma. These pages are “medium” fidelity and have flexible elements that can be hidden or shown to work in different complexities. You can use the “Prototype” tab in the righthand sidebar to get started with interactive prototype projects. When you’re ready to present, simply hit the play button in the top right.
```
Comment (ashygee) [inline pages/design/tools/figma.mdx]:
```suggestion
We’ve developed a prototyping toolkit that includes a set of page templates and components to help you get started with exploration and prototyping in Figma. These pages are “medium” fidelity and have flexible elements that can be hidden or shown to work in different complexities. You can use the “Prototype” tab in the righthand sidebar to get started with interactive prototype projects. When you’re ready to present, simply hit the play (▶️) button in the top right.
```

**Comment by ZainKho** [comment]
Styles and components crosslink in ‘Team Library’ section should automatically link to the respective sections, but won’t in the current site because Primer Design is not built on Primer Blueprints. When this moves to Doctocat, the functionality should be there. For now, I’ve used ‘a’ tags to link to the sections.

File: pages/design/tools/figma.mdx [hunk 4]
```diff
@@ -1,6 +1,110 @@
...
+   <Flex alignItems='center' flexDirection='row' mt='4' mb='4'>
+    <Box width={12/12}>
+      <BorderBox height="250px" overflow='hidden'>
+       <img src="https://user-images.githubusercontent.com/6846673/61730055-6ddf6000-ad2d-11e9-92ce-cd14fccc6851.png"/>
+      </BorderBox>
+    </Box>
+   </Flex>
+</Text>
+
+This page can help you learn how to use:
```
Comment (emilybrick) [inline pages/design/tools/figma.mdx]: This page can help you become for familiar with

File: pages/design/tools/figma.mdx [hunk 5]
```diff
@@ -1,6 +1,110 @@
...
+* [Team Library](#Library)
+* [Styles](#Styles)
+* [Components](#Components)
+* [Prototyping and Wireframing](#Prototyping)
+* [Collaboration](#Collaboration)
+
+## <a name="Library"></a>Team Library
+GitHub has a Figma team that contains all of GitHub's design assets. In Figma, [styles](#Styles) and [components](#Components) can be reused across multiple files and projects with the [team library](https://help.figma.com/article/29-team-library) feature. With team libraries, you can enable and disable components styles within certain files. This makes it easy for designers to easily access Primer components from anywhere.
```
Comment (emilybrick) [inline pages/design/tools/figma.mdx]:
```suggestion
GitHub has a Figma team that contains all of our design assets. In Figma, [styles](#Styles) and [components](#Components) can be reused across multiple files and projects with the [team library](https://help.figma.com/article/29-team-library) feature. With team libraries, you can enable and disable components styles within certain files. This makes it easy for designers to easily access Primer components from anywhere.
```

File: pages/design/tools/figma.mdx [hunk 6]
```diff
@@ -1,6 +1,110 @@
...
+      <BorderBox overflow='hidden'>
+       <img width="100%" src="https://user-images.githubusercontent.com/6846673/61732332-3f17b880-ad32-11e9-8e18-bfbfc0267d81.gif"/>
+      </BorderBox>
+      <Text textAlign='center' color='gray.6'>Enable and disable libraries of components from across your team.</Text>
+    </Box>
+   </Flex>
+</Text>
+
+## <a name="Styles"></a>Styles
+Primer uses Figma styles to maintain a consistent look and feel across all of GitHub's UI. We have provided styles for color variables as well as our typography.
```
Comment (emilybrick) [inline pages/design/tools/figma.mdx]: should link to the Figma styles resource here? "Primer uses [Figma styles](#).."

Comment (emilybrick) [inline pages/design/tools/figma.mdx]:
```suggestion
Primer uses Figma styles to maintain a consistent look and feel across all of our UI. We have provided styles for color variables as well as our typography.
```

File: pages/design/tools/figma.mdx [hunk 7]
```diff
@@ -1,6 +1,110 @@
...
+
+Color styles are used to keep colors consistent across different products. Different type styles are used based on the importance and actionability of the text.
```
Comment (emilybrick) [inline pages/design/tools/figma.mdx]:
```suggestion
```

I would remove this bit, it seems too generic to be useful atm. Think in the future we could flesh this out a bit more, but given the deadline I'd cut it.

File: pages/design/tools/figma.mdx [hunk 8]
```diff
@@ -1,6 +1,110 @@
...
+      </BorderBox>
+      <Text textAlign='center' color='gray.6'>Some components have multiple states, which you can toggle in the layer list.</Text>
+    </Box>
+   </Flex>
+</Text>
 
-## Components
+
+## <a name="Prototyping"></a>Prototyping and Wireframing
+We’ve developed a prototyping toolkit that includes a set of page templates and components to help you get started with exploration and prototyping in Figma. These pages are “medium” fidelity and have flexible elements that can be hidden or shown to work in different complexities. You can use the “Prototype” tab in the righthand sidebar to get started with interactive prototype projects. When you’re ready to present, simply hit the play (▶️) button in the top right.
```
Comment (emilybrick) [inline pages/design/tools/figma.mdx]:
```suggestion
We’ve developed a prototyping toolkit that includes a set of page templates and components to help you get started with exploration and prototyping in Figma. These pages have flexible elements that can be hidden or shown to work in different fidelities. You can use the “Prototype” tab in the righthand sidebar to get started with interactive prototype projects. When you’re ready to present, simply hit the play (▶️) button in the top right.
```

File: pages/design/tools/figma.mdx [hunk 9]
```diff
@@ -1,6 +1,110 @@
...
+       <img src="https://user-images.githubusercontent.com/6846673/61732575-bfd6b480-ad32-11e9-8e77-5a8c28b9bd88.png"/>
+      </BorderBox>
+    </Box>
+   </Flex>
+</Text>
+
+Duplicate this [Figma file](https://www.figma.com/file/ex9hnSDJTr8fG24cTMyRJzHD/prototyping-pages?node-id=0%3A1) to your own project to use these templates for prototyping or exploration.
+
+## <a name="Collaboration"></a>Collaboration
+All Hubbers have read-only access to all GitHub design files and edit access to files within their team. You can leave [Figma comments](https://help.figma.com/article/44-comments) to provide feedback within Figma and check [issues](https://help.github.com/en/articles/about-issues) to track decisions and progress. The design team works with incremental correctness, so share early and share often!
```
Comment (emilybrick) [inline pages/design/tools/figma.mdx]:
```suggestion
All Hubbers have read-only access to all GitHub design files and edit access to files within their team. You can leave [Figma comments](https://help.figma.com/article/44-comments) to provide feedback and use [issues](https://help.github.com/en/articles/about-issues) to document design exploration, track decisions and progress. The design team values [incremental correctness](https://github.com/github/product-design/blob/master/principles-and-practices.md#incremental-correctness), so share early and share often.
```

File: pages/design/tools/figma.mdx [hunk 10]
```diff
@@ -1,6 +1,110 @@
...
+
+## <a name="Resources"></a>Resources
+Check out these places to learn more about Figma and GitHub:
+- [Figma Design Documentation](https://help.figma.com/)
+- [GitHub takes its collaborative culture to a new level](https://www.figma.com/blog/github-case-study/)
+- [GitHub Primer Design](https://primer.style/design)
+
+List of current Primer Components:
```
Comment (emilybrick) [inline pages/design/tools/figma.mdx]: We talked irl about this, but I think the list of components could come first here

**Pull request merged**